### PR TITLE
Create parthdev.json

### PR DESCRIPTION
**Pull request by TheInfernitex**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [x] The file is in the `domains` folder and is in the JSON format.
- [x] The file's name is all lowercased and alphanumeric. 
- [x] You have completed your website. 
- [x] The website is reachable.  
- [x] The CNAME record doesn't contain `https://` or `/`.  
- [x] There is sufficient information at the `owner` field.  

## Website Link/Preview

image

### CODE REVIEW

File: domains/parthdev.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+    "owner": {
+        "username": "TheInfernitex"
```
Comment (DEV-DIBSTER) [inline domains/parthdev.json:3]: Email or OWL field is required here.

File: domains/parthdev.json [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+    },
+    "record": {
+        "CNAME": "TheInfernitex.github.io"
```
Comment (DEV-DIBSTER) [inline domains/parthdev.json:6]: 404 GitHub Page - Also flagged as dangerous by browser.

**Review by DEV-DIBSTER — CHANGES_REQUESTED** [review]

File: domains/parthdev.json (hunk 1 above)
Comment (TheInfernitex) [reply domains/parthdev.json:3]: Oh, Okay! I will add it. Thanks!

File: domains/parthdev.json (hunk 2 above)
Comment (TheInfernitex) [reply domains/parthdev.json:6]: My profile page shows 404 : https://github.com/TheInfernitex ?
One of my sites hosted on github pages does show up as dangerous though. I have not found the reason yet.

**Pull request closed without merging**